### PR TITLE
Add venture docs

### PR DESCRIPTION
**Pull request by wirtzdan**
I wrote down our basic process and reasoning behind building our own products. Open for improvement.

### CODE REVIEW

File: config/site-config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   primaryColor: "#153cff",
   copyright: "Made by Crisp Studio",
   asideMenu: [
-    { title: "Homepage", link: "https://crisp.studio" },
```
Comment (leolabs) [inline config/site-config.js:11]: I'd put this in a separate PR because it has nothing to do with the venture docs.

File: content/03-ventures/01-purpose.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Purpose
+
+## What is a Venture?
+
+The venture is _a risky or daring journey or undertaking_. For us, a venture is, when we develop a digital product on our own, that is useful for others and generates monthly recurring revenue. Most of the time, this is something we feel is missing in the market and that we are eager to bring to people.
```
Comment (leolabs) [inline content/03-ventures/01-purpose.md]:
```suggestion
The venture is _a risky or daring journey or undertaking_. For us, venturing means developing a digital product that is useful for others and generates monthly recurring revenue. Most of the time, this is something we feel is missing in the market and that we are eager to bring to people.
```

File: content/03-ventures/01-purpose.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Purpose
+
+## What is a Venture?
```
Comment (leolabs) [inline content/03-ventures/01-purpose.md]: I'd use the Chicago Manual of Style for the capitalization of titles. See [title.sh](https://title.sh) for an easy tool to generate them.

```suggestion
## What Is a Venture?
```

File: content/03-ventures/01-purpose.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+The venture is _a risky or daring journey or undertaking_. For us, a venture is, when we develop a digital product on our own, that is useful for others and generates monthly recurring revenue. Most of the time, this is something we feel is missing in the market and that we are eager to bring to people.
+
+## Practice what we preach
```
Comment (leolabs) [inline content/03-ventures/01-purpose.md]:
```suggestion
## Practice What We Preach
```

File: content/03-ventures/01-purpose.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+When we develop our own products, we can experiment with methods and technology in a safe environment. This helps us to grow the experience and confidence in our competences and to stay up to date.
+
+## The process
```
Comment (leolabs) [inline content/03-ventures/01-purpose.md]:
```suggestion
## The Process
```

File: content/03-ventures/02-pitches.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# Pitches
+
+A pitch is **detailed written presentation of a product idea**. If submitted at the right time, pitches will be included and discussed at the next Ideation Breakfast. It is important that the pitches stand alone and need no further explanation. Therefore the pitch should be clearly written and answer the most important questions. Each title is only a working title and can be changed in the actual project.
```
Comment (leolabs) [inline content/03-ventures/02-pitches.md]:
```suggestion
A pitch is a **detailed written presentation of a product idea**. If submitted at the right time, pitches will be included and discussed at the next Ideation Breakfast. It is important that the pitches stand alone and need no further explanation. Therefore the pitch should be clearly written and answer the most important questions. Each project title is only a working title and can be changed in the actual project.
```

File: content/03-ventures/02-pitches.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+
+The idea for writing pitches originates out of the book "[Thinking Fast and Slow](https://en.wikipedia.org/wiki/Thinking,_Fast_and_Slow)" by Daniel Kahneman. Kahnmann argues that the standard practice of open discussion gives to much weight to opinions of those who speak early and assertively. For us, this is very beneficial to make sure we value the diversity of knowledge and treat different opinions in the same manner.
```
Comment (leolabs) [inline content/03-ventures/02-pitches.md]:
```suggestion
The idea for writing pitches originates from the book "[Thinking Fast and Slow](https://en.wikipedia.org/wiki/Thinking,_Fast_and_Slow)" by Daniel Kahneman. Kahneman argues that "the standard practice of open discussion gives too much weight to the opinions of those who speak early and assertively [...]". For us, this is beneficial to make sure we value the diversity of knowledge and treat different opinions equally.
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
+# Ideation Breakfast
+
+The purpose of the ideation breakfast is to discuss and decide on the execution of internal product ideas. Ideas can be submitted in the form of pitches to be evaluated at the ideation breakfast. After the ideation breakfast, the chosen product idea can be scheduled, planned and executed.
+
+## Formulated document
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
## Formulated Document
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+Ideas are not presented in a spoken form but in a written pitch. The pitch needs to be written out in continuous text and should be self-explanatory without having further information. The structure can begin with a summary of the idea, followed by a description of the problem that led to the idea and then the solution. The deadline for submitting pitches is until the eve of each Ideation Breakfast. The deadline for submitting pitches is the evening before the ideation breakfast.
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
Ideas are not presented in a spoken form but in a written pitch. The pitch needs to be self-explanatory without having further information. The structure can begin with a summary of the idea, followed by a description of the problem that led to the idea and then the solution. The deadline for submitting pitches is the evening before the Ideation Breakfast.
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+[Example Pitch](https://www.notion.so/0727184bf5b840afa2fa4ea0c85ee864)
+
+## Read session
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
## Read Session
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+Before we go into the discussion of ideas, we take the time to read all pitches from front to back. This helps to make sure, that everyone is on the same page, understands the product idea in detail and had time to think about it. Everyone can take notes and questions that can be addressed later.
+
+## Questions
+
+After all have read through the pitches, the author of the pitch has to answer open questions. The answers help to deepen the understanding of the product idea and diminish uncertainty.
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
After everyone has read through the pitches, the author of the pitch can answer open questions. The answers help to deepen the understanding of the product idea and diminish uncertainty.
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+## Classify Impact / Effort
+
+In order to decide which product idea will make it into the next cycle, it is best to arrange the ideas according to impact and effort. The impact could include both economic potential and the impact of the problem.
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
In order to decide which product idea will make it into the next cycle, we arrange the ideas according to impact and effort. The impact could include both economic potential and the impact of the problem.
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+What the process looks like this:
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
The process looks like this:
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Draw an Impact / Effort Scale
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]: Regarding the whole impact/effort scale, it is more like a matrix or grid, so there are only four available options and no in-between:

- High impact / high effort
- Low impact / high effort
- High impact / low effort
- Low impact / low effort

See this example:
impact-effort-matrix-1

With this information in mind, we should re-think the seven steps you described below.

File: content/03-ventures/03-ideation-breakfast.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+2. Write the name of one product idea on a post-it
+3. Move your hand over the middle of the impact scale and ask, "Is the impact higher or lower?"
+4. The moderator should then move the note up or down until the others stop saying higher or lower
+5. Same procedure with the Effort scale
+6. Repeat the process for all other ideas
+7. The moderator should mark all ideas in the top left corner with a contrast dot
+
+## Prioritizing ideas
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
## Prioritizing Ideas
```

File: content/03-ventures/03-ideation-breakfast.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+
+If all ideas are hanging on the wall and have been discussed can be voted with sticky dots. The ideas are then sorted on the wall according to most votes.
```
Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]:
```suggestion
When all ideas are hanging on the wall and have been discussed, we vote for them with sticky dots and sort on the wall by the number of votes.
```

Comment (leolabs) [inline content/03-ventures/03-ideation-breakfast.md]: What happens if two or more ideas have the same amount of votes?

**Review by leolabs — CHANGES_REQUESTED** [review]
Most of the changes are only grammar/spelling changes, but we need to re-think some parts of the process (see comments).

File: content/03-ventures/01-purpose.md (hunk 3 above)
Comment (wirtzdan) [reply content/03-ventures/01-purpose.md]: Sounds useful to take this as a guideline. Is there are a good resource to scan through the Chicago Manual?

File: content/03-ventures/03-ideation-breakfast.md (hunk 14 above)
Comment (wirtzdan) [reply content/03-ventures/03-ideation-breakfast.md]: That's true. I copied this out of Notion. Going to change some wording here.

File: content/03-ventures/03-ideation-breakfast.md (hunk 16 above)
Comment (wirtzdan) [reply content/03-ventures/03-ideation-breakfast.md]: First, we put them on the same level. After that, we have to decide if we want to do another vote or declare a product idea just by discussing.

File: config/site-config.js (hunk 1 above)
Comment (wirtzdan) [reply config/site-config.js:11]: That's true. I will keep this in mind for next time and just push the change on this branch.

File: content/03-ventures/03-ideation-breakfast.md (hunk 16 above)
Comment (wirtzdan) [reply content/03-ventures/03-ideation-breakfast.md]: Great, much better wording. I just added a "them" after "and sort...". I felt it was missing there. 👍

Comment (wirtzdan) [reply content/03-ventures/03-ideation-breakfast.md]: I added this part to the docs.

File: content/03-ventures/01-purpose.md (hunk 3 above)
Comment (leolabs) [reply content/03-ventures/01-purpose.md]: You can read more about it here: http://www.chicagomanualofstyle.org/

**Review by leolabs — APPROVED** [review]
That looks good for now :)
